packages/server-side-render/src/constants.js

```diff
@@ -0,0 +1,81 @@
+export const ATTRIBUTE_PROPERTY = {
+	textColor: [ 'color', 'text' ],
+	backgroundColor: [ 'color', 'background' ],
+	gradient: [ 'color', 'gradients' ],
+	fontSize: [ 'typography', 'fontSize' ],
+	fontFamily: [ 'typography', '__experimentalFontFamily' ],
+	borderColor: [ '__experimentalBorder', 'color' ],
+};
+
+export const STYLE_PROPERTY = {
```

Comment:
This definition is based on the following:
https://github.com/WordPress/gutenberg/blob/b21766438d9bb0de9c04bc2494d7093fb12b9f25/packages/blocks/src/api/constants.js#L17